src/actions/AuthActions.js

```diff
@@ -15,6 +14,8 @@ import {
 import {FACEBOOK_AUTH} from '../../env';
 import {tokens, navKeys} from '../constants';
 
+import {persistor} from '../store';
```

Comment:
This is the same `persistor` object that we used in the `App.js` file. It's available to us anywhere that we'd like to use it in the app. You'll see below a handy use of it with the logout functionality.